sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/metadata.yaml

```diff
@@ -8,7 +8,10 @@ labels:
 scheduling:
   dag_name: bqetl_cloudflare_browser_market_share
   date_partition_parameter: dte
-  arguments: ["--date", "{{ds}}", "--cloudflare_api_token", "{{ var.value.cloudflare_auth_token}}"]
+  arguments: ["--date", "{{ds}}"]
+  secrets:
+  - deploy_target: CLOUDFLARE_AUTH_TOKEN
+    key: CLOUDFLARE_AUTH_TOKEN
```

Comment:
```suggestion
    key: bqetl_cloudflare_os_market_share__cloudflare_auth_token
```

Reply:
The suggested secret name doesn't match the `bqetl_cloudflare_browser_market_share` DAG name in this case.